packages/backend/src/utils/inferenceUtils.ts

```diff
@@ -138,7 +138,7 @@ export function generateContainerCreateOptions(
     HealthCheck: {
       // must be the port INSIDE the container not the exposed one
       Test: ['CMD-SHELL', `curl -sSf localhost:8000/docs > /dev/null`],
-      Interval: 1_000_000_000 * 15, // 15s
+      Interval: 1_000_000_000 * 5, // 15s
```

Comment:
```suggestion
      Interval: 1_000_000_000 * 5, // 5s
```
or maybe have a constant SECOND being 1_000_000_000 and then you use 5 * SECONDS (no need to have the comment that will be become obsolete / invalid at some point)

Reply:
Yes good idea, I created as you suggest a SECOND constant and removed the comments